gallery/src/main.scss

```diff
@@ -15,12 +20,6 @@
   width: 400px;
 }
 
-.gallery-example {
-  pre {
-    font-size: var(--nx-font-size-s);
```

Comment:
IMO Inconsolata is too small at this size.  Additionally it turns out this was affecting our `nx-pre` example which was definitely not intended.